tests/post/steps/test_volume.py

```diff
@@ -1,10 +1,39 @@
+import re
 from urllib3.exceptions import HTTPError
 
+import kubernetes.client
+from kubernetes.client import CustomObjectsApi
 from kubernetes.client import StorageV1Api
 from kubernetes.client.rest import ApiException
 import pytest
-from pytest_bdd import scenario, then, parsers
+from pytest_bdd import given, when, scenario, then, parsers
+import yaml
 
+from tests import utils
+
+
+# Constants {{{
+
+DEFAULT_VOLUME = """
```

Comment:
If there is a default volume we might want to avoid passing it in the scenarios themselves.

Reply:
Nope, two different use cases here.

You want to create a volume, and that's the point of the test?
=> then pass a YAML

Using a templated stuff for that will just give unreadable long line like:
`And I create a Persistent volume titled 'test-volume' of type 'sparseLoopDevice' with storageclass 'random-storageclass' of size '1Gi' for the 'bootstrap' node with a random device type 'rawSparseDevice'`
So, please no…

On the other hand, when you just need a volume to exists (because you want delete it or whatever) then we can use an hardcoded default one.